### ISSUE TITLE
Ergonomics improvement of the field « EVENT DESCRIPTION»

### ISSUE THREAD

**Issue by carolinverset** [hzi-braunschweig/SORMAS-Project#2415]
### Feature Description

Within the EVENT module, the current SORMAS version includes a text field “Event description” to formalize useful data concerning event description, comprehension and investigation.

### Problem Description

Users are informed of this size limitation only once entry is validated. Number of characters entered can’t be controlled throughout data entry.
Furthermore, as text is displayed on two lines only, users have to play with the lift to complete and read this zone.

### Proposed Change

It would thus be necessary to:
1)	Implement dynamic display of this zone to make it correspond to the text’s size.
2)	Indicate size limitation  by :

- Blocking data entry when the user reaches the maximum capacity of the field. This modification should be first implemented for the event formulary. It could further on be generalized to all fields of this type within SORMAS application ;
- Displaying the number of characters already entered or the number of characters left.

### Additional Information
screenshot 

![image](https://user-images.githubusercontent.com/65449450/86209543-153a5900-bb73-11ea-8f73-e61d73708674.png)

**Comment by bernardsilenou** [comment]
@MateStrysewskeSym @markusmann-vg  This has been implemented already by French team and we had a review yesterday. Awaiting pull request soon

**Comment by carolinverset** [comment]
@xavier-calland possible pour toi ?

**Comment by StefanKock** [comment]
>     * Blocking data entry when the user reaches the maximum capacity of the field. This modification should be first implemented for the event formulary. It could further on be generalized to all fields of this type within SORMAS application ;

This approach cuts off data when doing copy paste without noticing the user, so it will lead to data loss.

**Comment by MateStrysewskeSym** [comment]
I agree with Stefan and have created a follow-up issue for it: #2457
